packages/types/src/api.ts

```diff
@@ -384,4 +385,13 @@ export interface Endpoints {
   // Tags endpoints
   findTags: Endpoint<RequestPost<'/tags', string>, TagsResponse>;
   tags: Endpoint<RequestGet<'/tags'>, TagsResponse>;
+  // Zoom endpoints
+  zoomAuthorize: Endpoint<
```

Comment:
Good job with the endpoints specification!

This way we document through typescript all the endpoints. In the future we shall integrate this with the backend responses as well.

The `requests` on the web allows us to modify how `request` function signature.